File: Python/백준/Bronze/28417. 스케이트보드/스케이트보드.py
import sys

a = int(sys.stdin.readline())
score = []
for i in range(a):
    aa = list(map(int,sys.stdin.readline().split()))
    bb = aa[:2]
    cc = aa[2:]
    total = max(bb)+max(cc)
    cc.remove(max(cc))
    total+=max(cc)
    score.append(total)
sys.stdout.write(str(max(score)))

File: C#/백준/Bronze/28417. 스케이트보드/스케이트보드.cs
using System;

namespace q{
    class w{
        static void Main(String[] args){
            int a = int.Parse(Console.ReadLine());
            int max = 0;
            for (int i=0;i<a;i++){
                int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                int score = Math.Max(aa[0],aa[1]);
                aa[0]=-1; aa[1]=-1;
                Array.Sort(aa);
                score+=aa[6]+aa[5];
                max = max>score?max:score;
            }
            Console.WriteLine(max);
        }
    }
}